frontend/src/scenes/App.tsx

```diff
@@ -19,6 +19,20 @@ import { router } from 'kea-router'
 import { CommandPalette } from 'lib/components/CommandPalette'
 import { UpgradeModal } from './UpgradeModal'
 import { teamLogic } from './teamLogic'
+import posthog from 'posthog-js'
+import * as Sentry from '@sentry/browser'
+
+posthog.init((window as any).JS_POSTHOG_API_KEY || 'sTMFPsFhdP1Ssg', { api_host: (window as any).JS_POSTHOG_HOST })
```

Comment:
So this doesn't act as previous code does.

Previously, when users opted out of capturing posthog would not exist on the window for them/get init. Now we attempt to initialize posthog against app (?) even if user is opted out. We should not do that.

Reply:
Yep, good catch!